crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -1529,11 +1541,23 @@ mod tests {
 
         let now = std::time::Instant::now();
         converter
-            .execute_operation("5", "sleep", "sleep_one")
+            .execute_operation(
+                "5",
+                "sleep",
+                "sleep_one",
+                tokio::time::Duration::from_secs(0),
+                tokio::time::Duration::from_secs(0),
```

Comment:
Using 0 seconds as a timeout makes this test useless.

Indeed the point is to check that the converter returns immediately despite the operations take some time to run. But the timeout force the command to return immediately. So the test might be green but for bad reasons.